curp/src/client.rs

```diff
@@ -618,19 +624,20 @@ where
     /// # Panics
     ///   If leader index is out of bound of all the connections, panic
     #[inline]
-    #[instrument(skip_all, fields(cmd_id=%cmd.id()))]
+    #[instrument(skip_all)]
```

Comment:
Shall we add more information about the command in this instrument?

Reply:
I have no idea which infomation except id to instrument. The propose id will be `debug!` in the `fast_round` & `slow_round`

Reply:
IMO, we should include whether this Propose request is taking the fast_path or slow_path. This is because both fast_path and slow_path invoke fast_round and slow_round. Just based on this, we cannot tell whether a propose request is taking the fast_path or slow_path.

Reply:
I'll handle it in client refactor

Reply:
Please add an issue to track it.